### PR TITLE
Use "read_text()" so the file handler will be closed

### PR DESCRIPTION
**Pull request by arossert**
Addressing #13

### CODE REVIEW

File: cloud_detect/providers/alibaba_provider.py [hunk 1]
```diff
@@ -43,6 +43,6 @@ def check_vendor_file(self):
         self.logger.debug('Checking Aliobaba vendor file')
```
Comment (kshivakumar) [inline cloud_detect/providers/alibaba_provider.py:43]: Can you fix the typo - "Aliobaba"

File: cloud_detect/providers/azure_provider.py [hunk 2]
```diff
@@ -46,6 +46,6 @@ def check_vendor_file(self):
         self.logger.debug('Checking Azure vendor file')
         do_path = Path(self.vendor_file)
         if do_path.is_file():
```
Comment (kshivakumar) [inline cloud_detect/providers/azure_provider.py:48]: Replace `do_path` with `ms_path`. I forgot to update that in my PR.

**Comment by arossert** [comment]
@kshivakumar Can you please review?

**Comment by arossert** [comment]
@kshivakumar Following step 4 in the "How to contribute" (bug me until it gets merged and published.) 😃 
Can we get this published?

**Review by kshivakumar — APPROVED** [review]
LGTM. Please fix the "DCO check".

Only @dgzlopes(owner) can merge the PR.

**Comment by arossert** [comment]
@dgzlopes can you please look at this one?

**Comment by arossert** [comment]
@dgzlopes any chance this can be merged?

**Comment by arossert** [comment]
Thanks for the merge

**Pull request merged**